vector-im/element-android
# New Layout - Allows you to open an invite by clicking its cell

## Type of change


## Content

Brings back a missing feature that allows you to open an invite by clicking its cell

## Motivation and context

Fixes lost functionality

## Screenshots / GIFs

<img width="300" src="https://user-images.githubusercontent.com/20701752/188851255-41630057-0f93-4a6b-86fa-2ed558a73a10.png" />

## Tests



- Receive an invite (you can do this by DMing your account with a new test account)
- Open the invites page
- Click on the cell of the invite outside of the buttons
- See that you are on the screen as the screenshot above

## Tested devices

- OS version(s): Android 12

## Checklist
